backend/btrixcloud/colls.py

```diff
@@ -1009,24 +1007,10 @@ async def list_collection_all(
     @app.get(
         "/orgs/{oid}/collections/$all",
         tags=["collections"],
-        response_model=Dict[str, List[CrawlFileOut]],
```

Comment:
Should re-add or update response_model

Reply:
Updated in: https://github.com/webrecorder/browsertrix/pull/2516/commits/b08309eed52804a9877de8e9ed1c4100023fee7b